solutions/devsprint-leonardo-santos-1/FinanceAppTests/Service/FinanceServiceTests.swift

```diff
@@ -0,0 +1,62 @@
+import XCTest
+@testable import FinanceApp
+
+class FinanceServiceTests: XCTestCase {
+
+    func testFetchUserProfile() {
```

Comment:
Como sugestão de nomenclatura, tente dar nomes mais descritivos para os seus testes, dessa forma a leitura fica muito mais fácil. Tente pensar em `teste_métodoTestado_quandoOCenarioForSucesso_deveRetornarXResultado`, porém em inglês. Nesse teste abaixo, sugiro colocar um nome parecido com este:  
```suggestion
    func test_fetchUserProfile_givenUrl_shouldCallNetworkClientToPerformRequest() {
```